### ISSUE TITLE
Question about hosting model

### ISSUE THREAD

**Issue by tesar-tech** [CuriousDrive/BlazingChat#34]
First of all, thanks for this great resource!

I hope this is the right place to ask a question about your app. 

The app is separated to WebApi and Client part. When one creates the default blazor template with VS, apps (api and client) are somehow glued together. When you run the api part it will also start (host) the client. Basically you can run the app with one `dotnet run` command. On the other hand, Blazing chat needs to run the parts separately. Also the hosting "model" on azure has 2 resources (static web app and web app), but it is possible to run whole app in just the web app resource... What are the reasons for that? I have few ideas:
  - For the mobile app, you would need to separate the app anyway. Maybe not, but it feels wrong to have one of the clients (the blazor web app) glued to web api which will be used by other client apps (droid, ios)...
  - The development is kind of faster. For example when using `dotnet watch run`, it scans and rebuilds only the appropriate part of the app (client or api).
  - It could deploy only the part that is actually updated. But when you edit something in client app it will trigger both gh actions, thus redeploy the web api unnecessary. Is there an option to deploy just the client? (or am I mistaken here?)
  - Anything else??

Thank you!

**Comment by CuriousDrive** [comment]
Thanks for asking the question & thanks for the pull request too :)

I had the same questions Jan. The main reason why I decided to separate client and web API is because I wanted to deploy client on the Azure Static Web App. It gives me free SSL certificate and I can also map my domain name to it. Azure Web app free tier does not let you set up custom domain for FREE. 

I wanted to deploy the whole app for FREE, that was my main goal. If you are fine spending some money then what you have mentioned above should work just fine.

I hope I answered your question,
